{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/tasks.py

```diff
@@ -0,0 +1,33 @@
+{% include 'misc/header.py' %}
+"""Tasks for {{ cookiecutter.project_name }}."""
+
+from __future__ import absolute_import, print_function
+
+from celery import shared_task
+from invenio_db import db
+from invenio_records_files.api import Record
+from invenio_records_files.models import RecordsBuckets
+
+
+@shared_task(ignore_result=True)
+def update_record_files_by_bucket(bucket_id):
+    """Given a bucket id, dump its files in the record metadata."""
+    record_bucket = \
+        RecordsBuckets.query.filter_by(bucket_id=bucket_id).first()
+    if record_bucket:
+        record = Record.get_record(record_bucket.record_id)
+        record.files.flush()
+        record.commit()
+        # it might fail with `sqlalchemy.orm.exc.StaleDataError`
```

Comment:
Can you catch the error here to make it explicit and simply do a continue inside - just to make it explicit and avoid that errors are reported in celery which is actually expected behaviour

Reply:
Will fix in the next PR: https://github.com/inveniosoftware/cookiecutter-invenio-instance/pull/200